Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -911,7 +916,9 @@ private extension SPNoteListViewController {
             for note in notes {
                 SPObjectManager.shared().restoreNote(note)
             }
+            SPTracker.trackNoticeActionMultipleTrashUndo()
         }))
+        SPTracker.trackNoticeNotesTrashed()
```

Comment:
We could align the name of both. Something like:

```
let multiple = notes.count > 1
SPTracker.trackNoticeTrash(multiple: multiple)
```

Same for the Undo? maybe Undo could even be a parameter (undoing). Not sure about the last part!

Reply:
I kinda feel about this one the same way that I do with the link copied notices.  We could combine these track events into a single method, but the only time this notice would be presented is if we were doing a multi edit delete, and the notice is specific to that action.  

If we are tracking the presentation of the notice, I feel like multi delete and undo should have their own track methods.